duo-client/pom.xml

```diff
@@ -80,6 +80,20 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-checkstyle-plugin</artifactId>
+        <version>3.1.1</version>
+        <configuration>
+          <configLocation>google_checks.xml</configLocation>
```

Comment:
I don't see a google_checks.xml file added to this commit. Does it need to be, or is it somehow automatically downloaded/used?